CHANGELOG.rst

```diff
@@ -1,6 +1,12 @@
 Changes by Version
 ==================
 
+0.22.0-RC1 (2017-10-05)
+-------------------
+- Bump OpenTracing version to 0.31.0-RC1 (https://github.com/jaegertracing/jaeger-client-java/pull/246)
```

Comment:
This change is on a branch while the others are in master. Perhaps we should add a `(on branch v0.22.0)` so that people aren't confused.